### DISCUSSION TITLE
ansible-lint - install ansible modules/collections

### DISCUSSION THREAD

**Discussion by scaarup** [super-linter/super-linter#4187] [Q&A]
Hi all.
My ansible-lint is failing with "ansible-playbook --syntax-check" on a many of my playbooks. This is because the docker container does not have the ansible modules installed, which I am referring to.
How do I work around that?

Thanks

**Comment by ferrarimarco** [comment]
Ansible-lint supports requirements files (https://ansible.readthedocs.io/projects/lint/usage/#dependencies-and-requirements).

Did you try that? If that doesn't work, can you please open a new issue for this?

**Comment by s-m-p** [comment]
There's a way to "fake" the linter into thinking the module is available. Add this to your ANSIBLE_CONFIG_FILE:
`mock_modules:`
`  - community.general.mail`

That technique resolves it for me.
